apps/admin/backend/src/app.ts

```diff
@@ -393,10 +400,18 @@ function buildApi({
       return addCastVoteRecordReportResult;
     },
 
-    clearCastVoteRecordFiles(): void {
+    async clearCastVoteRecordFiles(): Promise<void> {
       const electionId = loadCurrentElectionIdOrThrow(workspace);
       store.deleteCastVoteRecordFiles(electionId);
       store.setElectionResultsOfficial(electionId, false);
+      await logger.log(
```

Comment:
I really don't expect logging to fail. It makes sense to me to do fire-and-forget, but I don't anticipate it would make a UX difference either way. @carolinemodic might have thoughts.